### PR TITLE
Add a recipe for sherlockml-dataclean

### PR DESCRIPTION
**Pull request by shwinnn**
Adding a recipe to install the Jupyter notebook extension and python package sherlockml-dataclean.

All listed recipe maintainers have agreed.

### CODE REVIEW

**Comment by shwinnn** [comment]
Hi, are any further changes required to get this merged?

**Comment by shwinnn** [comment]
gentle bump @conda-forge/staged-recipes

File: recipes/sherlockml-dataclean/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (CJ-Wright) [inline recipes/sherlockml-dataclean/meta.yaml]: Could this be noarch?

File: recipes/sherlockml-dataclean/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - jupyter
```
Comment (CJ-Wright) [inline recipes/sherlockml-dataclean/meta.yaml]: Is this a build dependency?

File: recipes/sherlockml-dataclean/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    A Jupyter plugin for interactive cleaning of pandas DataFrames.
+
+extra:
+  recipe-maintainers:
+    - shwinnn
+    - acroz
+    - neb42
+    - zblz
+    - srstevenson
+    - pbugnion
```
Comment (scopatz) [inline recipes/sherlockml-dataclean/meta.yaml]: @acroz @neb42 @zblz @srstevenson @pbugnion are you willing to be maintainers?

File: recipes/sherlockml-dataclean/meta.yaml (hunk 1 above)
Comment (CJ-Wright) [reply recipes/sherlockml-dataclean/meta.yaml]: totally missed that, sorry!

**Comment by shwinnn** [comment]
I've spoken to the listed maintainers in person, happy to wait for them to confirm on here if that's required before merging.

File: recipes/sherlockml-dataclean/meta.yaml (hunk 2 above)
Comment (shwinnn) [reply recipes/sherlockml-dataclean/meta.yaml]: @pbugnion I based this bit on your gmaps recipe, is it correct?

**Comment by scopatz** [comment]
Yeah, they need to confirm here prior to merging

**Comment by zblz** [comment]
Yes, I'm willing to be a maintainer.

>

File: recipes/sherlockml-dataclean/meta.yaml (hunk 2 above)
Comment (pbugnion) [reply recipes/sherlockml-dataclean/meta.yaml]: This recipe runs `jupyter nbextension {enable,disable} ...` in the pre-unlink/post-unlink scripts, so it definitely needs to be there at runtime, but presumably not at build-time?

**Comment by pbugnion** [comment]
I'm happy to be a maintainer. @shwinnn I suggest removing other maintainers (besides `zblz` and myself) for now. That's enough to prevent a single point of failure. People can just add themselves by submitting a PR to a recipe that we will have commit rights to, so the overhead of adding maintainers is low.

File: recipes/sherlockml-dataclean/meta.yaml (hunk 3 above)
Comment (srstevenson) [reply recipes/sherlockml-dataclean/meta.yaml]: Yep.

**Comment by acroz** [comment]
I'm happy to be a maintainer if useful - sorry for delay.

**Comment by scopatz** [comment]
Thanks All!

**Pull request merged**